nvflare/fuel/hci/cmd_arg_utils.py

```diff
@@ -68,3 +71,82 @@ def get_usage(self) -> str:
         usage_output = buffer.getvalue().split("\n", 1)[1]
         buffer.close()
         return usage_output
+
+
+def process_targets_into_str(targets: List[str]) -> str:
+    if not isinstance(targets, list):
+        raise SyntaxError("targets is not a list.")
+    if not all(isinstance(t, str) for t in targets):
+        raise SyntaxError("all targets in the list of targets must be strings.")
```

Comment:
We are raising the same SyntaxError here. Instead of call "raise SyntaxError()" multiple places, we can generate the error message from different place, and then call the "raise SyntaxError(error_message)" in one place. This maybe a little cleaner.

Reply:
This was from old code (just moved from FLAdminAPI to here to be reused). It's probably not worth the time to do what you suggested.